docs/source/intro/installation.rst

```diff
@@ -1,16 +1,652 @@
+
+#. There is a `known issue <https://github.com/Microsoft/WSL/issues/856>`_ in WSL Ubuntu 18.04 where the timezone is not configured correctly out-of-the-box, which may cause problem for the database.
+   The following command can be used to re-configure the time zone:
+
+   .. code-block:: console
+
+      $ sudo dpkg-reconfigure tzdata
+
+#. The file open limit may need to be raised using ``ulimit -n 2048`` (default is 1024), when running tests.
+   You can check the limit by using ``ulimit -n``.
+
+   .. hint:: This may need to be run every time the system starts up.
+
+It may be worth considering adding some of these commands to your ``~/.bashrc`` file, since some of these settings may reset upon reboot.
+
+.. admonition:: Further Reading
+   :class: title-icon-read-more
+
+   For using WSL as a developer, please see the considerations made in our `wiki-page for developers <https://github.com/aiidateam/aiida-core/wiki/Development-environment#using-windows-subsystem-for-linux-wsl>`_.
+
+.. _install/virtual_environments:
+
+Virtual environments
+====================
+
+AiiDA depends on a number of third party python packages, and usually on specific versions of those packages.
+In order not to interfere with third party packages needed by other software on your system, we **strongly** recommend isolating AiiDA in a virtual python environment.
+
+.. admonition:: Additional Information
+   :class: seealso title-icon-read-more
+
+   A very good tutorial on python environments is provided by `realpython.com <https://realpython.com/effective-python-environment>`__.
+
+`venv <https://docs.python.org/3/library/venv.html>`__ is module included directly with python for creating virtual environments.
+To create a virtual environment, in a given directory, run:
+
+.. code-block:: console
+
+   $ python3 -m venv /path/to/new/virtual/environment/aiida
+
+The command to activate the environment is shell specific (see `the documentation <https://docs.python.org/3/library/venv.html#creating-virtual-environments>`__.
+With bash the following command is used:
+
+.. code-block:: console
+
+   $ source /path/to/new/virtual/environment/aiida/bin/activate
+
+To leave or deactivate the environment, simply run:
+
+.. code-block:: console
+
+    (aiida) $ deactivate
+
+.. admonition:: Update install tools
+   :class: tip title-icon-tip
+
+   You may need to install ``pip`` and ``setuptools`` in your virtual environment in case the system or user version of these tools is old
+
+   .. code-block:: console
+
+      (aiida) $ pip install -U setuptools pip
+
+If you have `Conda`_ installed then you can directly create a new environment with ``aiida-core`` and (optionally) Postgres and RabbitMQ installed.
+
+.. code-block:: console
+
+   $ conda create -n aiida -c conda-forge python=3.7 aiida-core aiida-core.services pip
+   $ conda activate
+   $ conda deactivate aiida
+
+.. _directory_location:
+
+Isolating AiiDA environments
+----------------------------
```

Comment:
Note that this content is already largely present [elsewhere](https://aiida.readthedocs.io/projects/aiida-core/en/docs-revamp/howto/installation.html#isolating-multiple-instances). It might be ok to keep a minimal version here, but I would try to just link to the other section as much as possible and prevent duplication

Reply:
Thats exactly what I'm doing right now 😁

Reply:
Fixed in 9144b03